server/routes/tenants.js

```diff
@@ -3,6 +3,22 @@ const router = express.Router();
 
 const Tenant = require("../models/tenant")
 
+// const tenants = [];
+
+// idk if i did this right, might have to debug?
+// gets tenant that matches tenant ID provided
+router.get('/:tenantId', async(req, res, next) => {
+    const tenantId = req.params.tenantId;
+    try {
+    const foundTenant = await Tenant.find({tenantId: tenantId});
```

Comment:
MongoDB stores the id field as `_id`, so using `tenantId` to filter would not work. If you want to filter using something like `find` or `findOne`, you're going to have to make sure that the fields match the schema.

I would also suggest using the `findById` method instead: https://mongoosejs.com/docs/api/model.html#Model.findById()

It would probably look something like this (test this out first if you go with it): 
```suggestion
    const foundTenant = await Tenant.findById(tenantId);
```